<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>580</width>
      <height>570</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>580</width>
      <height>570</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>580</width>
      <height>570</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>性能</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>内存</string>
      </property>
     </widget>
     <widget class="QLabel" name="memused">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>使用:</string>
      </property>
     </widget>
     <widget class="QLabel" name="memfree">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>空闲:</string>
      </property>
     </widget>
     <widget class="QLabel" name="memtotal">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>总和:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>利用率:</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="cpu_customplot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>561</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="mem">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>内存使用率:</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="mem_customplot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>561</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>581</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>581</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>交换</string>
      </property>
     </widget>
     <widget class="QLabel" name="swapused">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>360</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>使用:</string>
      </property>
     </widget>
     <widget class="QLabel" name="swapfree">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>空闲:</string>
      </property>
     </widget>
     <widget class="QLabel" name="swaptotal">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>360</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>总和:</string>
      </property>
     </widget>
     <widget class="QLabel" name="swap">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>交换率:</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="swap_customplot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>561</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>进程信息</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_proc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="proc">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>进程数:</string>
      </property>
     </widget>
     <widget class="QLabel" name="run">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>运行数:</string>
      </property>
     </widget>
     <widget class="QLabel" name="sleep">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>120</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>睡眠数:</string>
      </property>
     </widget>
     <widget class="QLabel" name="zombie">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>170</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>僵尸数:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="kill">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>460</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>结束进程</string>
      </property>
     </widget>
     <widget class="QPushButton" name="proc_refresh">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>500</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_process">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>420</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>启动进程</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="procname_line">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>240</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="proc_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>270</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>系统信息</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu_name">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>411</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU名称:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu_type">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>411</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU类型:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu_frequency">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>401</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU频率:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cache_size">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>411</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cache大小:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>操作系统信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="OS_type">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>421</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>操作系统类型:</string>
      </property>
     </widget>
     <widget class="QLabel" name="OS_version">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>421</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>操作系统版本:</string>
      </property>
     </widget>
     <widget class="QLabel" name="gcc">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>431</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>GCC编译器:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reboot">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>400</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>重启</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shutdown">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>400</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>关机</string>
      </property>
     </widget>
     <widget class="QLabel" name="boot_time">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>431</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>系统启动时间:</string>
      </property>
     </widget>
     <widget class="QLabel" name="run_time">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>431</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>系统运行时间:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>关于</string>
     </attribute>
     <widget class="QLabel" name="my_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="my_xuehao">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="my_class">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>班级：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionshutdown">
   <property name="text">
    <string>shutdown</string>
   </property>
  </action>
  <action name="actionreboot">
   <property name="text">
    <string>reboot</string>
   </property>
  </action>
  <action name="action123">
   <property name="text">
    <string>123</string>
   </property>
  </action>
  <action name="action1234">
   <property name="text">
    <string>1234</string>
   </property>
  </action>
  <action name="action12345">
   <property name="text">
    <string>12345</string>
   </property>
  </action>
  <action name="action123456">
   <property name="text">
    <string>123456</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
